docs/concepts/trust_levels.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 🎚 Trust Levels
+description: What level of "community trust" does SourceCred depend on
+---
+
+SourceCred is intended to be used by communities to divy up rewards, in the
+form of cred and grain. As such, SourceCred needs to be resilient to gaming by
+people who explicitly (but perhaps covertly) attempt to maximize their rewards
+at all other costs.
+
+The kind of gaming that will happen depends on the ambient level of trust and
+social cohesion within the community. A small community of like-minded and
+highly aligned people might see very little gaming, with most people explicitly
+working together to make sure everyone is fairly credited. On the other hand, a
+larger and more anonymous community may have many selfish actors who just want
+to increase their scores without regard for the broader mission.
+
+We've developed the conept of "Trust Levels" to try to categorize communities
+along this dimension of trust, so that we can reason about which communities
+SourceCred is ready to support, and what kinds of problems communities might
+encounter along the way.
+
+  - **Trust level 3** (TL3): Collaborative. This is the highest trust level.
+    It is taken as [common knowledge][common-knowledge] that everyone in the
+    community is acting in good faith. People feel comfortable reflecting on
+    their own incentives: “am I subconsciously skewing the cred to benefit
+    myself?”; “how have my motivations changed over time?”.
+    When asking the same questions of others, we trust that the responses are
+    thoughtfully and truthfully given. Discussions of accidental or
+    subconscious gaming are treated as precious data to be shared and analyzed,
+    never as evidence of any kind of wrongdoing.
+
+    *As an example, imaging group of friends launching a startup together, with
+    a bold vision of improving the world.*
+
+  - **Trust level 2** (TL2): Wary. The community has a core backbone of
+    contributors that are focused on common goals, but some people may be more
+    focused on optimizing their own rewards. Primarily distinguished from TL3
```

Comment:
Not sure I understand the sentence "primarily distinguished..."
Also, it is a sentence fragment.